### PR TITLE
[14.0][IMP] account_financial_risk: consider multicurrency

### PR DESCRIPTION
**Pull request by AaronHForgeFlow**
And some fix.

### CODE REVIEW

**Review by JoanMForgeFlow — APPROVED** [review]
LG!

File: account_financial_risk/models/account_invoice.py [hunk 1]
```diff
@@ -82,4 +83,43 @@ def action_post(self):
                     )
                     .action_show()
                 )
-        return super().action_post()
+        return super().post()
```
Comment (carlosdauden) [inline account_financial_risk/models/account_invoice.py]: ¿Why this change?
¿`def action_post(self):` and `return super().post()`?

Please, include this changes:https://github.com/OCA/credit-control/commit/4bd8fcd1f1cdc8120b354f0054782ecb24659948

**Review by carlosdauden — CHANGES_REQUESTED** [review]

File: account_financial_risk/models/account_invoice.py (hunk 1 above)
Comment (carlosdauden) [reply account_financial_risk/models/account_invoice.py]: This wizard call to method `_post`:
https://github.com/odoo/odoo/blob/37481a9f797e6dc519405aef1e6196a9e76a9203/addons/account/wizard/account_validate_account_move.py#L22
therefore method `_post` should be extended, and not `action_post`, to the warning is displayed when several invoices are validated

Comment (carlosdauden) [reply account_financial_risk/models/account_invoice.py]: Ping @psugne https://github.com/OCA/credit-control/commit/5d95c65e3c569426696b7d0400d51566e4105e14#diff-75cb51ce626ac1a0a6744302a700a18a6df917b7f76a8bb9a27a4d363c28e8bbR62

Comment (AaronHForgeFlow) [reply account_financial_risk/models/account_invoice.py]: thank you @carlosdauden that was a mistake doing the cherrypick. I included 4bd8fcd now. But I am setting this to WIP, I have to test it again.

File: account_financial_risk/models/res_partner.py [hunk 2]
```diff
@@ -30,80 +31,80 @@ class ResPartner(models.Model):
...
-        string="Include Other Account Unpaid Amount",
+        string="Include Other Overdue Amounts",
         help="Full risk computation.\n"
         "Residual amount of move lines not reconciled with distinct "
         "account that is set as partner receivable and date maturity "
         "exceeded, considering Due Margin set in account settings.",
     )
     risk_account_amount_unpaid_limit = fields.Monetary(
-        string="Limit Other Account Unpaid Amount",
+        string="'Limit Other Overdue Amounts",
```
Comment (MiquelRForgeFlow) [inline account_financial_risk/models/res_partner.py:101]:
```suggestion
        string="Limit Other Overdue Amounts",
```

**Comment by AaronHForgeFlow** [comment]
This is actually a customization. Closing.
